boards/arc/em_starterkit/doc/board.rst

```diff
@@ -174,122 +181,55 @@ Parity:    None
 
-For Linux, you will need:
+Boot up the SOC by pressing the "C" button. Be sure the digilent cable
+is attached from your host to the EM Starter Kit board.
 
-* arc_gnu_2016.03_ide_linux_install.tar.gz
-* arc_gnu_2016.03_prebuilt_elf32_be_linux_install.tar.gz
-
-You can untar these into any directory. Suggestion: /usr/local/arc.  Follow the
-instructions for how to set up to use these tools.  You will need your path
-changed to refer to the bin directories for these so that these cross
-development tools can be found.
+Debugging
+==========
 
-It is also useful to create a bash script to launch openocd, since if you are
-like me, you will find it hard to remember the arguments. Here is the one I use,
-placed in a file called ocd.
+Build and debug the application with the following commands:
 
 .. code-block:: console
 
-  #!/bin/bash
-  openocd -c 'gdb_port 3333' -s $ARCGNU_IDE/share/openocd/scripts -f board/snps_em_sk_v2.2.cfg
+   $ cd <my app>
+   $ make BOARD=em_starterkit debug
 
-  #where the environment variable ARCGNU_IDE refers to the install dir of the
-  #openocd IDE
-
-This command to openocd uses port 3333 with gdb client, and provides the board
-cfg file for the ARC EM Starter Kit.
+At this point you can do your normal debug session. Set breakpoints and then
+'c' to continue into the program.
 
-I have also found it useful to have a script, named debug.sh, to provide all the
-arguments to gdb:
+Launch the debug server on the emsk:
 
 .. code-block:: console
 
-   #!/bin/bash
-   arc-elf32-gdb \
-    -ex "target remote :3333" \
-    -ex "load" \
-    -ex "break __memory_error" \
-    -ex "break _SysFatalErrorHandler" \
-    -ex "set remotetimeout 2000" \
-    outdir/em_starterkit/zephyr.elf
+   $ make BOARD=em_starterkit debugserver
 
-The target remote and load will attach to openocd and load the elf file into
-memory. You can also set breakpoints on CPU exception handlers, or fatal
-error handlers.
-
-Now to use these two scripts is easy. Boot up the SOC by pressing the "C" button.
-Be sure the digilent cable is attached from your host to the EM Starter Kit
-board.
-
-In any terminal window, invoke the "ocd" script first. It should establish
-contact with the board and output many messages. For example:
+Connect to the debug server at the emsk from a second console:
 
 .. code-block:: console
 
-   Open On-Chip Debugger 0.9.0-dev-g90a4ff5 (2016-05-04-15:37)
-   Licensed under GNU GPL v2
-   For bug reports, read
-   http://openocd.sourceforge.net/doc/doxygen/bugs.html
-   adapter speed: 5000 kHz
-   Info : clock speed 5000 kHz
-   Info : JTAG tap: arc-em.cpu tap/device found: 0x200044b1 (mfg: 0x258, part: 0x0004, ver: 0x2)
-   Info : JTAG tap: arc-em.cpu tap/device found: 0x200044b1 (mfg: 0x258, part: 0x0004, ver: 0x2)
-   target state: halted
-   target state: halted
+   $ cd <my app>
+   $ $ZEPHYR_SDK_INSTALL_DIR/sysroots/x86_64-pokysdk-linux/usr/bin/arc-zephyr-elf/arc-zephyr-elf-gdb outdir/em_starterkit/zephyr.elf
```

Comment:
you could make this line shorter so it doesn't wrap by breaking the line at a space and put in a backslash